onionshare_gui/mode/share_mode/__init__.py

```diff
@@ -199,25 +206,25 @@ def stop_server_custom(self):
             self._zip_progress_bar = None
 
         self.filesize_warning.hide()
-        self.downloads_in_progress = 0
-        self.downloads_completed = 0
-        self.update_downloads_in_progress()
+        self.history.in_progress_count = 0
+        self.history.completed_count = 0
+        self.history.update_in_progress()
         self.file_selection.file_list.adjustSize()
 
     def cancel_server_custom(self):
         """
         Stop the compression thread on cancel
         """
         if self.compress_thread:
-            self.common.log('OnionShareGui', 'cancel_server: quitting compress thread')
+            self.common.log('ShareMode', 'cancel_server: quitting compress thread')
             self.compress_thread.quit()
 
     def handle_tor_broke_custom(self):
         """
         Connection to Tor broke.
         """
         self.primary_action.hide()
-        self.info_widget.hide()
+        self.info_label.hide()
```

Comment:
I think this should be removed or commented out same as Receive mode (info label e.g '7 files, 120 kb' above FileList) is OK to stay)

Reply:
Good idea